plugins/outputs/circonus/circonus.go

```diff
@@ -308,7 +310,13 @@ func (c *Circonus) buildTexts(defaultDest *cgm.CirconusMetrics, m cua.Metric) in
 	tags := c.convertTags(m.Origin(), m.OriginInstance(), m.TagList())
 	for _, field := range m.FieldList() {
 		mn := strings.TrimSuffix(field.Key, "__value")
-		dest.SetTextWithTags(mn, tags, field.Value.(string))
+		//c.Log.Infof("MN=%s Tags=[%v] %v=%v[%T]\n", mn, tags, field.Key, field.Value, field.Value)
+		switch v := field.Value.(type) {
```

Comment:
This is a hack around metrics with multiple fields due to SeriesGrouper having mixed types.  A real solution is underway so feel free to take this out.